index.html

```diff
@@ -8,8 +8,11 @@
 </head>
 <body>
     <h1 id="title">Michael Tribute Page</h1>
+
     <figure id="img-div">
+
         <img alt src="https://www.grammy.com/sites/com/files/styles/news_detail_header/public/ftr-e40-michaeljackson-thumb-ig.jpg?itok=wqWRccRj">
```

Comment:
Maybe try to use proper spacing in-between attributes to not trip up the linter errors! - Simple Suggestion